apps/rates-service/src/internals/rates/UpdateDb.ts

```
@@ -114,52 +114,73 @@ export async function ensureLatestFxRate(now: number) {
+  // don't check them individually
+  // because our entry point implements
+  // backoff/retry logic.  If the ensure
+  // is called again it won't matter because it's already
+  // up to date.
+  const r = await Promise.allSettled([
+    ensureLatestCoinRate(now),
+    ensureLatestFxRate(now),
+  ]);
+
+  // Collect any rejections to throw
+  const rejections = r.filter(r => r.status === "rejected") as PromiseRejectedResult[];
```

ESLint

[error] Expected parentheses around arrow function argument.

(arrow-parens)